src/stores/sessions.js

```diff
@@ -36,6 +40,56 @@ export const useSessionsStore = defineStore('sessions', {
           }
         }
       }
+    },
+    async fetchToken () {
+      const requestBody = JSON.stringify(this.currentSession)
+      const response = await fetchApiCall({
+        url: 'http://rti-bridge-dev.lco.gtn/login',
+        method: 'POST',
+        body: JSON.parse(requestBody)
+      })
+      if (!this.getTokenForCurrentSession) {
+        this.sessionTokens[this.currentSessionId] = response.token
+      }
+    },
+    async fetchStatus () {
+      const token = this.getTokenForCurrentSession
+      if (!token) {
+        await this.fetchToken()
+      }
+
+      const response = await fetchApiCall({
+        url: 'http://rti-bridge-dev.lco.gtn/session_status',
+        method: 'GET',
+        header: { Authorization: `Token ${token}` }
+      })
+      this.currentStatus = response.session_status
+    },
+    startPolling () {
+      this.stopPolling()
+
+      const poll = async () => {
+        await this.fetchStatus()
+        const time = calculateTime(this.currentSession)
+        let nextInterval = 60000
+        // 10 minutes (600 seconds) before session start poll every second -- time is arbitrary
```

Comment:
Why do we poll more often before the session starts and less after?

Reply:
Yeah I think swapping these two polling intervals would work better - e.g. every 10s before the session, and every second during. We want the session to end as quickly as possible once the bridge is returning INACTIVE for the session status.

Reply:
gotcha! changing that now : )